scss/main.scss

```diff
@@ -169,10 +166,25 @@ button[type="submit"] {
   color: $form-text-light;
 }
 
+// project-list
+
+.project-list {
+  .projects {
```

Comment:
If you use `row flex-items-center` as the classes instead, it does the same thing (with a minor difference in margin), and then you don't need this CSS. (By consistently using the bootstrap row/column classes, you avoid the margin issue.)